.github/workflows/pr.yml

```diff
@@ -177,12 +183,64 @@ jobs:
       - name: Untar artifacts
         shell: bash
         run:  tar xfz target.tgz
-      - name: setup node
-        shell: bash
-        run: (cd ui/client; npm ci)
       - name: Prepare docker
         shell: bash
         run: echo "version in ThisBuild := \"$GITHUB_SHA\"" > version.sbt && ./ciRunSbt.sh dist/docker:publishLocal
       - name: Test docker
         shell: bash
-        run: cd ./demo/docker && echo "NUSSKNACKER_VERSION=$GITHUB_SHA" > .env && ./testQuickstart.sh
+        run: cd ./demo/docker && echo "NUSSKNACKER_VERSION=ghactionstest-$GITHUB_SHA" > .env && ./testQuickstart.sh
```

Comment:
Any env from  file `.env` is used only when specific env is not declared. So enough will be only declare `NUSSKNACKER_VERSION=ghactionstest-$GITHUB_SHA`.

Reply:
:+1:  true, thanks